src/components/calcite-action/calcite-action.scss

```diff
@@ -1,246 +1,230 @@
 :host {
   @extend %component-host;
-  @extend %component-spacing;
-  display: flex;
-  background-color: transparent;
+  @apply flex bg-transparent;
 }
 
 :host([disabled]) {
-  pointer-events: none;
+  @apply pointer-events-none;
 }
 
 .button {
+  @apply bg-foreground-1
+    border-none
+    cursor-pointer
+    fill-color-3
+    flex
+    focus-base
+    items-center
+    justify-start
+    m-0
+    relative
+    text--2h
+    text-color-3;
   font-family: inherit;
-  display: flex;
-  color: var(--calcite-ui-text-3);
-  fill: var(--calcite-ui-text-3);
-  background-color: var(--calcite-ui-foreground-1);
-  margin: 0;
-  justify-content: flex-start;
-  align-items: center;
-  border: none;
   width: auto;
-  cursor: pointer;
   text-align: unset;
-  position: relative;
-  font-size: theme("fontSize.-2");
-  @include focus-style-base();
-  &:hover,
-  &:focus {
-    background-color: var(--calcite-ui-foreground-2);
-    color: var(--calcite-ui-text-1);
-    fill: var(--calcite-ui-text-1);
+  &:hover {
+    @apply bg-foreground-2 text-color-1 fill-color-1;
   }
 
   &:focus {
-    background-color: var(--calcite-ui-foreground-2);
-    color: var(--calcite-ui-text-1);
-    fill: var(--calcite-ui-text-1);
-    @include focus-style-inset();
+    @apply bg-foreground-2 text-color-1 fill-color-1 focus-inset;
   }
 
   &:active {
-    background-color: var(--calcite-ui-foreground-3);
+    @apply bg-foreground-3;
   }
 
   .icon-container {
-    min-width: 1rem;
-    min-height: 1rem;
-    margin: 0;
-    justify-content: center;
-    align-items: center;
-    display: flex;
-    pointer-events: none;
+    @apply flex
+      items-center
+      justify-center
+      m-0
+      pointer-events-none;
+    min-width: theme("spacing.4");
+    min-height: theme("spacing.4");
```

Comment:
Also here, `items-center` is vague.  Does it apply to `align-items` or `justify-items`?

Reply:
Ah, yeah, the learning curve on the name choices is a bit unfortunate. The brevity is coming from the inline use
case. In practice, it becomes second nature after some use.


| Property | Utility |
| -------- | ------- |
| align-items | items-* |
| align-content | content-* |
| align-self | self-* |
| justify-content | justify-* |

Where `*` is the value. In the case of a hypenated value, the last word is used e.g.:

`justify-content: space-evenly;` -> `justify-evenly`

Reply:
Cool, thanks for the info on that, I'll be sure to reference back to this!